waspc/examples/todoApp/sample.env.client

```diff
@@ -0,0 +1,2 @@
+#REACT_APP_API_URL=http://localhost:4041
```

Comment:
Default for backend is 3001, so maybe let's set this to default value, that is kind of usual?

Reply:
Good idea, that'll be less confusing